src/crate/crash/repl.py

```diff
@@ -274,21 +275,40 @@ def create_buffer(cmd, history_file):
 
 
 def loop(cmd, history_file):
+
+    def session_toolbar(cli):
+        tokens = []
+        if cmd.is_conn_avaliable():
+            active_servers = cmd.connection.client.active_servers
+            active_servers = [n.replace('http://', '') for n in active_servers]
+            active_servers = [n.replace('https://', '') for n in active_servers]
```

Comment:
could use generator expressions instead of list comprehensions to avoid the copying